pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[build-system]
+requires = ["flit_core >=2,<3", "requests = 2.23.0"]
+build-backend = "flit_core.buildapi"
+
```

Comment:
Is there a minimum required version of Python3? 

`requires-python` configuration. https://flit.readthedocs.io/en/latest/flit_ini.html

We might as well put in keywords to make this package discoverable.

Reply:
Tracked in #35